vms/event/templates/event/edit.html

```diff
@@ -100,76 +100,50 @@
                         </div>
                     </div>
 
-				{% if form.country.errors %}
-                    <div class="form-group has-error">
-                        <label class="col-md-2 control-label">{% trans "Country" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event Country{% endblocktrans %}" type="text" name="country" value="{% if form.country.value %}{{ form.country.value }}{% endif %}">
-                            <p class="help-block">
-                                <strong>
-                                    {% for error in form.country.errors %}
-                                        {{ error }}
-                                    {% endfor %}
-                                </strong>
-                            </p>
-                        </div>
-                    </div>
-                {% else %}
-                    <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "Country" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event Country{% endblocktrans %}" type="text" name="country" value="{% if form.country.value %}{{ form.country.value }}{% endif %}">
-                        </div>
-                    </div>
-                {% endif %}
+                    
```

Comment:
In this page, initially it should show the already selected city, country and state when the event was created.

Reply:
Also the filter functionality.

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/22369767/41200738-1155d004-6cc8-11e8-8f3b-57217592462e.png)

Reply:
Did you commit this? I can't see the changes